src/tokens/TokenDetails.tsx

```diff
@@ -68,6 +69,9 @@ export default function TokenDetailsScreen({ route }: Props) {
   const actions = useActions(token)
   const tokenDetailsMoreActionsBottomSheetRef = useRef<BottomSheetRefType>(null)
   const localCurrencySymbol = useSelector(getLocalCurrencySymbol)
+  const usePriceHistoryFromBlockchainApi = getFeatureGate(
```

Comment:
Add your wallet to `use_price_history_from_blockchain_api` in statsig to test locally.